dashboard/test/controllers/pairings_controller_test.rb

```diff
@@ -0,0 +1,94 @@
+require 'test_helper'
+
+class PairingsControllerTest < ActionController::TestCase
+  include Devise::TestHelpers
+
+  setup do
```

Comment:
That said, consider moving `sign_in create(:user)` to the `setup` (saving the `user` for the methods that need to reference it).